samples/boards/esp32/wifi_station/prj.conf

```diff
@@ -0,0 +1,26 @@
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP32=y
+CONFIG_HEAP_MEM_POOL_SIZE=98304
+CONFIG_THREAD_CUSTOM_DATA=y
+CONFIG_DYNAMIC_INTERRUPTS=y
```

Comment:
How are those 2 options used? If Wifi driver requires them, then maybe they should be selected by it in Kconfig?

Reply:
Yes, wifi driver needs it.